plugins/view-resources/src/components/list/ListCategory.svelte

```diff
@@ -283,7 +283,7 @@
       try {
         const newRank = calcRank(prev, next)
         if ((dragItem.doc as DocWithRank)?.rank !== newRank) {
-          ;(update as any).rank = newRank
+          (update as any).rank = newRank
```

Comment:
these changes break CI formatting check, `;` is important here and similar places